src/logic/safe/transactions/gasNew.js

```diff
@@ -118,8 +118,12 @@ export const estimateSafeTxGas = async (
               gasLimit: txGasEstimation + dataGasEstimation + additionalGas,
             },
             (error, res) => {
+              // res.data check is for OpenEthereum/Parity revert messages format
+              const isEstimationSuccessful =
+                !error && (res !== '0x' || (res.data === 'string' && res.slice(9) !== '0x'))
```

Comment:
so if `res` is an object with `{ data: "blabla" }` and `error` is `null` then `res !== '0x'` will evaluate to `true` and therefore `(res.data === 'string' && res.slice(9) !== '0x')` will not be evaluated, right?

Reply:
do we have any tests for this logic here?

Reply:
@rmeissner oh nice catch with the res.data. thing

and we don't have any tests

Reply:
how would we test this? I can see that in safe-contracts it skips the test https://github.com/gnosis/safe-contracts/blob/f0ad68f9cf7977d3e8adab1ae88bb6e5b25b8c76/test/safeTxGasEstimation.js#L60